optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -300,9 +299,8 @@ def pre_attn_forward(
                 past_key_value = (self.k_cache.get_shape(), self.v_cache.get_shape())
             else:
                 if past_key_value is None:
-                    past_key = torch.zeros(key_states.shape, dtype=self.k_proj.weight.dtype, device=key_states.device)
-                    past_value = torch.zeros(key_states.shape, dtype=self.k_proj.weight.dtype, device=key_states.device)
-                    past_key_value = (past_key, past_value)
+                    # Return list instead of tuple
```

Comment:
this won't work for fp8 flow, need to revert, and maybe add a comment why it was added at the first place (allocating the kv cache according to k_proj type which can be bf16/fp8

Reply:
@bgoldberg-habana  - Fixed.